src/main/scala/streaming/SampleStatusesHashtagPrinter.scala

```diff
@@ -0,0 +1,30 @@
+package streaming
+
+import com.danielasfregola.twitter4s.TwitterStreamingClient
+import com.danielasfregola.twitter4s.entities.Tweet
+
+
+object SampleStatusesHashtagPrinter extends App {
+
+  val client = TwitterStreamingClient()
+
+  def printHashtags(tweet: Tweet) = tweet.entities match {
```

Comment:
Function not used anywhere. Is this intentional?

Reply:
Yeah... well now it is :D